sbt-twirl/src/main/scala/play/twirl/sbt/TemplateProblem.scala

```diff
@@ -87,7 +88,7 @@ object TemplateProblem {
     }
 
     def apply(source: Option[File], codec: Codec): TemplateMapping = {
-      val lines = source.toSeq flatMap { file => IO.readLines(file, codec.charSet) }
+      val lines = source.toSeq flatMap { file => TwirlIO.readFileAsString(file, codec).split('\n') }
```

Comment:
You can fix it differently. I tried to do it as simple as possible, without breaking any API.

BTW, there is `stripLineEnd` method in `StringLike` class, which looks a little bit nicer than `stripSuffix`. It's used in SBT [here](https://github.com/sbt/sbt/blob/v0.13.11/compile/interface/src/main/scala/xsbt/DelegatingReporter.scala#L61).